scripts/compile_java_test_and_verify.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+mvn -B $EXTRA_FLAGS clean compile test | tee build.log
+
+tests_run=$(cat build.log  | grep "Tests run:" | tail -n 1 | sed 's/.*Tests run: \([0-9]*\).*/\1/')
+
+echo "DEBUG: tests_run = $tests_run"
+
+if [ -z "$tests_run" ]; then
+echo "WARNING: Could not determine the number of tests run."
+tests_run=0
```

Comment:
Should indent here.

Should we just exit here? If not, the output will be confusing:

```
WARNING: Could not determine the number of tests run.
ERROR: No tests were run!
```